WikiDrips/WikiRequest.swift

```diff
@@ -84,6 +84,6 @@ public class WikiRequest
             return nil
         }
 
-        return URLRequest(url: url)
+        return URLRequest(url: url, cachePolicy: .returnCacheDataElseLoad)
```

Comment:
Oops, I missed one instance of this.  I had previously fixed it on WikiDocViewController.  So an oversight, not a regression.